src/background/adblocker.js

```diff
@@ -167,23 +184,6 @@ export const setup = asyncSetup('adblocker', [
   ),
 ]);
 
-let ENABLE_FIREFOX_CONTENT_SCRIPT_SCRIPLET_INJECTION = false;
```

Comment:
The variable is used above that part of the code. I recommend to define variables on top of the code which uses them.